docs/marks.rst

```diff
@@ -132,7 +159,7 @@ The ``@pytest.mark.tier`` takes single number as an argument.
     def test_tier1()
         pass
 
-You can then run tests that are relevant only to the selected ticket:
+You can then run tests by tier:
```

Comment:
Tier section can be completely removed as we switched from tier to importance.

Reply:
AFAIK, tiers will be designated for different architectures. When we run our regression runs, tier0 will be x86_64, tier1, tier2 will be other architectures. For now, I'll remove it, but it may be added again later.